### PR TITLE
Fix Russian translation

### PR DESCRIPTION
**Pull request by GiovanniMIS**

### CODE REVIEW

**Comment by BluSunrize** [comment]
I have marked this as a draft PR, since it is obviously still WIP.

Please notify us when it's ready to be merged.

**Review by Smollet777 — APPROVED** [review]

File: src/main/resources/assets/immersiveengineering/lang/ru_ru.json [hunk 1]
```diff
@@ -734,24 +760,56 @@
   "block.immersiveengineering.sheetmetal_aluminum": "Алюминиевый листовой металл",
   "block.immersiveengineering.sheetmetal_lead": "Свинцовый листовой металл",
   "block.immersiveengineering.sheetmetal_silver": "Серебряный листовой металл",
-  "block.immersiveengineering.sheetmetal_nickel": "Никелевый листовой металл",
+  "block.immersiveengineering.sheetmetal_nickel": "Никилевый листовой металл",
   "block.immersiveengineering.sheetmetal_uranium": "Урановый листовой металл",
-  "block.immersiveengineering.sheetmetal_constantan": "Константановый листовой металл",
```
Comment (Smollet777) [inline src/main/resources/assets/immersiveengineering/lang/ru_ru.json]: Было правильно

File: src/main/resources/assets/immersiveengineering/lang/ru_ru.json [hunk 2]
```diff
@@ -734,24 +760,56 @@
   "block.immersiveengineering.sheetmetal_aluminum": "Алюминиевый листовой металл",
   "block.immersiveengineering.sheetmetal_lead": "Свинцовый листовой металл",
   "block.immersiveengineering.sheetmetal_silver": "Серебряный листовой металл",
-  "block.immersiveengineering.sheetmetal_nickel": "Никелевый листовой металл",
```
Comment (Smollet777) [inline src/main/resources/assets/immersiveengineering/lang/ru_ru.json]: Таки через "е"

**Comment by BluSunrize** [comment]
I can't review this, since I don't speak Russian, but if you think it's ready, I'll happily merge it.

**Review by Smollet777 — APPROVED** [review]

**Comment by BluSunrize** [comment]
Thank you =)

**Pull request merged**